docs/user-guide/dhcp-management.rst

```diff
@@ -0,0 +1,63 @@
+.. _dhcp-management:
+
+***************
+DHCP Management
+***************
+
+Cobbler can optionally help you manage a DHCP server. This feature is off by default.
+
+The following options are available for ``modules.dhcp.module``:
+
+* ``"managers.isc"``
+* ``"managers.dnsmasq"``
+
+Set ``manage_dhcp: true`` and ``manage_dhcp_v4`` or ``manage_dhcp_v6`` to ``true`` for this setting to take effect.
+
+This allows DHCP to be managed via "cobbler system add" commands, when you specify the MAC address and IP address for
+systems you add into Cobbler.
+
+You must configure the templates for your networking environment. Read the file and understand how
+the particular app work before proceeding.
+
+If you already have DHCP configuration data that you would like to preserve (such as DHCP that was manually configured earlier),
+insert the relevant portions of it into the template file, as running ``cobbler sync`` will overwrite your previous
+configuration.
+
+By default, Cobbler updates the DHCP configuration file each time you run ``cobbler sync``.
+Remember to use ``cobbler sync`` when you use this feature.
+
+isc DHCP
```

Comment:
```suggestion
``isc`` DHCP
```
Try to set names of services in codeblocks so they stand out. this is a personal style choice and you can disregard it when you want to.